apps/frontend/src/app/Data/Characters/Yoimiya/index.tsx

```diff
@@ -144,7 +144,11 @@ const atk_ = greaterEq(
     )
   )
 )
-const c1atk_ = equal(condC1, 'c1', percent(dm.constellation1.atk_))
+const c1atk_ = greaterEq(
```

Comment:
@nguyentvan7 just based on intuition, `condC1` should be locked behind the constellation check as well, right?